core/src/main/java/greencity/controller/OrderController.java

```diff
@@ -468,4 +447,48 @@ public ResponseEntity<HttpStatus> receivePaymentClient(
         }
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Controller for getting all Active Locations if user haven't made any order
+     * before. If user has made an order before controller returns info about tariff
+     * by which it was made Controller is used to get all active Locations if user
+     * want to change location to make an order
+     *
+     * @param changeLoc - optional param. If it is present controller will return
+     *                  info about locations
+     * @param uuid      - user's uuid
+     * @return {@link OrderCourierPopUpDto}
+     */
+    @ApiOperation(value = "Get all location where courier is working")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @GetMapping("/allLocations")
+    public ResponseEntity<OrderCourierPopUpDto> getAllActiveLocatins(
+        @RequestParam Optional<String> changeLoc,
+        @ApiIgnore @CurrentUserUuid String uuid) {
+        return ResponseEntity.status(HttpStatus.OK)
+            .body(ubsClientService.getInfoForCourierOrdering(uuid, changeLoc));
+    }
+
+    /**
+     * Controller for getting info about tariff for location.
+     *
+     * @param locationId - id of location
+     * @return {@link OrderCourierPopUpDto}
+     */
+    @ApiOperation(value = "Get tariff for location and courier")
+    @ApiResponses(value = {
+        @ApiResponse(code = 200, message = HttpStatuses.OK),
+        @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
+        @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
+        @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
+    })
+    @GetMapping("/tariffinfo-for-location/{locationId}")
+    public ResponseEntity<OrderCourierPopUpDto> getInfoAboutTariff(@Valid @PathVariable Long locationId) {
```

Comment:
once you have Valid annotation, please add appropriate Controller test to make sure API returns 400 on invalid input parameters/path variables